.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## RVM Version
```

Comment:
@pkuczynski Good call about missing `shell` info. About the order: I prefer to keep consistent with other tools and repositories: environment is always asked first. Anyway, not a strong feeling from my side.